lib/releaselib.sh

```diff
@@ -769,23 +783,10 @@ release::docker::release () {
       logecho "Release $docker_target:"
       logecho -n "- Pushing: "
       logrun -r 5 -s ${docker_push_cmd[@]} push "$registry/$docker_target"
-
-      # If we have a amd64 docker image. Tag it without -amd64 also
-      # and push it for compatibility with earlier versions
-      if [[ $arch == "amd64" ]]; then
-        legacy_docker_target="$binary:$version"
-        logecho "Release legacy $legacy_docker_target:"
-
-        logecho -n "- Tagging: "
-        logrun docker rmi "$registry/$legacy_docker_target" || true
-        logrun -r 5 -s docker tag "$registry/$docker_target" \
-                              "$registry/$legacy_docker_target" 2>/dev/null
-
-        logecho -n "- Pushing: "
-        logrun -r 5 -s ${docker_push_cmd[@]} \
-                       push "$registry/$legacy_docker_target"
-      fi
     done
+
+    # This command will push a manifest list: "${registry}/${binary}-ARCH:${version}" that points to each architecture depending on which platform you're pulling from
+    $MANIFEST_TOOL push from-args --platforms ${ml_platforms} --template ${registry}/${binary}-ARCH:${version} --target ${registry}/${binary}-ARCH:${version}
```

Comment:
is ARCH treated as a special value? should both the template and target contain ARCH?